flags.css

```diff
@@ -1,8 +1,56 @@
 /* Indonesia */
+.indonesiaFlag {
+    border: 2px solid;
+    height: 200px;
```

Comment:
Don't make a new flag container, use the .flagContainer that you already have

Reply:
make .flagContainer have a size, that way you can reuse it for most of your flags.  Otherwise it's a pain in the ass to do it over and over